server/plugin/utils.go

```diff
@@ -206,3 +206,41 @@ func (p *Plugin) IsAnyProjectLinked(mattermostUserID string) (bool, error) {
 func (p *Plugin) getConnectAccountFirstMessage() string {
 	return fmt.Sprintf(constants.ConnectAccountFirst, fmt.Sprintf(constants.ConnectAccount, p.GetPluginURLPath(), constants.PathOAuthConnect))
 }
+
+func (p *Plugin) ParseSubscriptionsToCommandResponse(subscriptionsList []serializers.SubscriptionDetails, channelID string) string {
+	var sb strings.Builder
+
+	if len(subscriptionsList) <= 0 {
```

Comment:
```suggestion
	if len(subscriptionsList) == 0 {
```

Reply:
there would be no chances for `<` but adding this keeps in safer side as we are considering values only > 0